artiq/coredevice/phaser.py

```diff
@@ -886,6 +958,30 @@ def trf_read(self, addr, cnt_mux_sel=0) -> TInt32:
         return self.trf_write(0x00000008 | (cnt_mux_sel << 27),
                               readback=True)
 
+    @kernel
+    def cal_vco(self):
+        """Start VCO calibration."""
+        self.trf_write(self.trf_mmap[1] | (1 << 31))
+
+    @kernel
+    def en_out(self, rf=1, lo=0):
+        """Enable the rf/lo outputs.
+
+        It is recommended to disable TRF outputs during VCO calibration.
+
+        :param rf: 1 to enable RF output, 0 to disable
+        :param lo: 1 to enable LO output, 0 to disable"""
+        data = self.trf_read(0xc)
+        delay(0.1 * ms)
+        # outputs disabled
+        data = data | (1 << 12) | (1 << 13) | (1 << 14)
+        # toggle if enabled
```

Comment:
Do you mean "clear to enable"?

Reply:
Yes, though these amount to the same thing here.